packages/snaps-controllers/src/cronjob/CronjobController.test.ts

```diff
@@ -92,7 +93,7 @@ describe('CronjobController', () => {
 
     const callActionMock = jest
       .spyOn(controllerMessenger, 'call')
-      .mockImplementation((method) => {
+      .mockImplementation((method, ..._) => {
```

Comment:
What's this for?

Reply:
Visual Studio Code is using new version of TS, and it's complaining that we're not accepting all the parameters of the `call`. This silences TypeScript in VSC, and when we're going to upgrade to newer version.